campaigns/rustreach/how-I-came-to-rust.md

```diff
@@ -0,0 +1,58 @@
+---
+title: How I came to Rust
+---
+
+# How I came to Rust
+
+The #rustreach program asked people to write down how they came to Rust and I'm happy to do that. Entering a community or a space is always a moment of orientation process and I hope I can give some insight into that.
+
+My path is highly individual, but yours will be, too. Own that. It's great.
+
+## Picking up Rust
+
+I am certainly an early adopter of Rust. I picked it up as a hobby in the early versions, Rust 0.4 was the first version I tried out and around version 0.6, I started writing notable pieces of software in it. That was in 2013. Rust - for me - was the chance to see a language being built. I had no interested in writing a programming language, but I was curious how the process works. I'm still amazsed on how agressively languages can change when compatibilty is no issue. Back then, _all Rust code in existence_ would break _regularly_.
```

Comment:
I suggest "no interested" -> "no interest" :)
And "amazsed on" -> "amazed at" :)